modules/globebrowsing/src/globerotation.cpp

```diff
@@ -87,6 +87,12 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    // This rotation class will orient the scene graph node in such a way that the y-axis
+    // points away from the provided globe, the x-axis points towards the globe's southern
+    // pole and the x-axis points in a western direction. Using this rotation generally
+    // means using the [GlobeTranslation](#globebrowsing_translation_globetranslation) to
+    // place the scene graph node at the same position for which the rotation is
+    // calculated.
```

Comment:
```suggestion
    // This `Rotation` will orient the scene graph node in such a way that the y-axis
    // points away from the provided globe, the x-axis points towards the globe's southern
    // pole and the z-axis points in a western direction. Generally, this rotation should be 
    // combined with a [GlobeTranslation](#globebrowsing_translation_globetranslation) to
    // place the scene graph node at the same position for which the rotation is calculated.
```
I think ? the x-axis was repeated twice. Place verify that the change is correct.